site/content/docs/5.1/components/orange-sticker.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: docs
+title: Orange sticker
+description: Use Boosted's custom stickers to inform people about new offers.
+group: components
+toc: true
+---
+
+## Overview
+
+Stickers have their own relative sizes with `rem` units. They also have their own pattern.
+
+## Examples
+
+<div class="row align-items-center my-5">
+  <div class="col col-12 col-lg-5 text-start">
+    <svg viewBox="0 0 140 140" class="sticker-lg sticker-one-line" aria-hidden="true" focusable="false">
```

Comment:
Might be a dump questions but why are we using SVG rather that a simple div with border-radius?

Something like:
```html
<div style="width: 280px; height: 280px; background-color: #f16e00; border-radius: 280px" class="d-flex align-items-center justify-content-center">
  <span style="font-weight: 700; font-size: 60px">New</span>
</div>
```